dynamic_rest/filters.py

```diff
@@ -460,96 +493,109 @@ def _filter_queryset(
         """
 
         is_root_level = False
-        if serializer:
-            if queryset is None:
-                queryset = serializer.Meta.model.objects
+        if model:
```

Comment:
It looks like most of the code in this function is now wrapped in `if serializer`. Would it make sense to copy just the code that's relevant for the `if model` case in a separate method?